br/pkg/lightning/config/config.go

```diff
@@ -1070,6 +1070,8 @@ type TikvImporter struct {
 	StoreWriteBWLimit       ByteSize `toml:"store-write-bwlimit" json:"store-write-bwlimit"`
 	// default is PausePDSchedulerScopeTable to compatible with previous version(>= 6.1)
 	PausePDSchedulerScope PausePDSchedulerScope `toml:"pause-pd-scheduler-scope" json:"pause-pd-scheduler-scope"`
+	// block size is in KiB
+	BlockSize int `toml:"block-size" json:"block-size"`
```

Comment:
I just checked that we want the configuration syntax like `RegionSplitSize`

```
block-size = '16KiB'
```

https://github.com/pingcap/tidb/issues/45037#issuecomment-1685512854

We should change the type to `ByteSize`, remove the "block size is in KiB" comment and remove `* units.KiB` in local.go line 876